### ISSUE TITLE
aria-labelledby on Dropdown is set to Dropdown itself if no id provided

### ISSUE THREAD

**Issue by IvanShastin** [microsoft/fluentui#21887]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen:
https://codesandbox.io/s/tv5h0?module=/example.js&file=/example.js

#### Actual behavior:

aria-labelledby is set to id of dropdown button and macOS voiceover does not read text, insteand says button

#### Expected behavior:
 instead should ignore and read text from span in dropdown

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

**Comment by gouttierre** [comment]
Thanks for filing this issue with us. To set expectations, the developers will review this issue once capacity allows.
@jurokapsiar- Would your team be able to confirm if this is a regression, or if this behavior is an issue?
